src/PrometheusPushGateway/PushGateway.php

```diff
@@ -25,12 +25,12 @@ class PushGateway
 
     /**
      * PushGateway constructor.
-     * @param string $address host:port of the push gateway
+     * @param string $address http|https://host:port of the push gateway
      * @param ClientInterface $client
      */
     public function __construct($address, ClientInterface $client = null)
     {
-        $this->address = $address;
+        $this->address = strpos($address, 'http') === false ? 'http://' . $address : $address;
```

Comment:
Could you please add a unit test for this? We should make sure that this isn't a breaking change.

Reply:
Of course, where are my manners? I added a test to cover the funky address logic.